packages/presentation/lib/src/app/widgets/app_navigation_bar.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:presentation/src/app/app_bloc.dart';
+import 'package:presentation/utils/images_container.dart';
+
+class AppNavigationBar extends StatelessWidget {
+  const AppNavigationBar({
+    Key? key,
+    required this.bottomNavIndex,
+    required this.bloc,
+  }) : super(key: key);
+  final AppBloc bloc;
+  final int bottomNavIndex;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      decoration: BoxDecoration(
+        border: Border(
+          top: BorderSide(
+            color: Theme.of(context).colorScheme.onPrimary,
+            width: 1.0,
+          ),
+        ),
+      ),
+      child: BottomNavigationBar(
+        currentIndex: bottomNavIndex,
```

Comment:
can we remove this property? seems that bottom navigation should handle it's state by itself. but if it is not possible - let's keep it